drivers/serial/uart_mcux_flexcomm.c

```diff
@@ -1067,6 +1117,42 @@ static int mcux_flexcomm_init(const struct device *dev)
 	return 0;
 }
 
+static uint32_t usart_intenset;
```

Comment:
This variable is not used outside the scope of `mcux_flexcomm_pm_action`, move it inside, also does not need to be static